README.asciidoc

```diff
@@ -369,6 +369,19 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Scripting]]
+Hints on scripted usage
+-----------------------
+If you want to run image-bootstrap from a shell script and capture its output
+to STDOUT/STDERR in a log file, be sure to run image-bootstrap in unbuffered
+mode, e.g.:
+
+`python -u image-bootstrap [params] 2>&1 | tee my.log`
```

Comment:
Let's either fully inline the command or make it stand out as a block as done elsewhere in this file. I'd recommend this syntax ..

```
------------------------------------------------------------------------------------------
python -u image-bootstrap [params] 2>&1 | tee my.log
------------------------------------------------------------------------------------------
```

.. (with a single blank line before and after) to match the rest of the file.

Reply:
Let's go with "elsewhere" style